x-pack/metricbeat/cmd/root.go

```diff
@@ -5,11 +5,14 @@
 package cmd
 
 import (
-	"github.com/elastic/beats/metricbeat/cmd"
+	"github.com/elastic/beats/filebeat/cmd"
```

Comment:
@sayden that also seems wrong

Reply:
Yeah, definitely that shouldn't be there

Reply:
I have just pushed a commit with this fixed